docs/developer/users.mdx

```diff
@@ -254,41 +271,58 @@ curl \
   http://localhost:8000/graphql/
 ```
 
-### Verifying a token
+### Verifying token
 
 To verify the token, use the following mutation:
 
 ```graphql {2}
 mutation {
-  tokenVerify(token: "<your-token>") {
-    payload
+  tokenVerify(token: "<your-access-token>") {
+    isValid
+    accountErrors {
+      field
+      code
+    }
   }
 }
 ```
 
+:::note
+You can verify both access tokens and refresh tokens.
+:::
+
 A successful response:
 
 ```json
 {
   "data": {
     "tokenVerify": {
-      "payload": {
-        "email": "admin@example.com",
-        "exp": 1566915066,
-        "origIat": 1566548900
-      }
+      "isValid": true,
+      "accountErrors": []
     }
   }
 }
 ```
 
-### Refreshing a token
+### Refreshing an access token
+
+The `tokenRefresh` mutation will generate a new access token when provided with a valid refresh token and a matching CSRF token.
+If the refresh token is not provided as an argument, the code will try to read it from a cookie set by the `tokenCreate` mutation.
+
+The mutation takes the following input fields:
+
+- `refreshToken`: the refresh token, if not specified will be read from browser cookies.
+- `csrfToken`: the CSRF token used to protect against cross-site resource forgery attacks.
```

Comment:
We need to update this section to reflect that only one of those is required. We should also provide separate examples for those two cases.

Reply:
Yes, sorry. I forgot about this.